xDrip/Components/Views/GlucoseCurrentInfoView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  GlucoseCurrentInfoView.swift
+//  xDrip
+//
+//  Created by Dmitry on 6/16/20.
+//  Copyright © 2020 Faifly. All rights reserved.
+//
+
+import UIKit
+@IBDesignable
+class GlucoseCurrentInfoView: UIView, NibLoadable {
+    @IBOutlet private weak var glucoseIntValueLabel: UILabel!
+    @IBOutlet private weak var glucoseDecimalValueLablel: UILabel!
+    @IBOutlet private weak var slopeArrowLabel: UILabel!
+    @IBOutlet private weak var lastScanTitleLabel: UILabel!
+    @IBOutlet private weak var lastScanValueLabel: UILabel!
+    @IBOutlet private weak var difTitleLabel: UILabel!
+    @IBOutlet private weak var difValueLabel: UILabel!
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupFromNib()
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
```

Comment:
If you init a view with frame (not with coder), the application will fail because of missing view. Replace the content of this initialiser with `fatalError("init with frame is not supported")`